scripts/types_generator/schema_aliases.yaml

```diff
@@ -2,464 +2,477 @@ alias:
+    - _PromptCreate_result_parameters_decoding_method
+    - _TextChatOutputCreateRequest_parameters_decoding_method
+    - _RequestChatConversationIdRetrieve_results_request_parameters_decoding_method
+    - _PromptIdUpdate_result_parameters_decoding_method
+    - _TextChatStreamCreateRequest_parameters_decoding_method
+    - _PromptRetrieve_results_parameters_decoding_method
+    - _PromptIdPatch_result_parameters_decoding_method
+    - _TextChatCreateRequest_parameters_decoding_method
+    - _PromptIdUpdateRequest_parameters_decoding_method
+    - _TextGenerationComparisonCreateRequest_request_parameters_decoding_method
+    - _UserPatch_result_generate_default_parameters_decoding_method
+    - _TextGenerationOutputCreateRequest_parameters_decoding_method
+    - _PromptCreateRequest_parameters_decoding_method
   TextGenerationParametersDecodingMethod:
-    - TextGenerationStreamCreateRequest_parameters_decoding_method
-    - TextGenerationCreateRequest_parameters_decoding_method
+    - _TextGenerationStreamCreateRequest_parameters_decoding_method
+    - _TextGenerationCreateRequest_parameters_decoding_method
   StopReason:
-    - TextGenerationCreate_results_stop_reason
-    - TextChatStreamCreate_results_stop_reason
-    - TextGenerationStreamCreate_results_stop_reason
-    - TextGenerationComparisonCreate_results_result_results_stop_reason
-    - TextChatCreate_results_stop_reason
-    - RequestChatConversationIdRetrieve_results_response_results_stop_reason
+    - _TextGenerationCreate_results_stop_reason
+    - _TextChatStreamCreate_results_stop_reason
+    - _TextGenerationStreamCreate_results_stop_reason
+    - _TextGenerationComparisonCreate_results_result_results_stop_reason
+    - _TextChatCreate_results_stop_reason
+    - _RequestChatConversationIdRetrieve_results_response_results_stop_reason
   TextGenerationLimit:
-    - TextGenerationLimitRetrieve_result
+    - _TextGenerationLimitRetrieve_result
   TextEmbeddingParameters:
-    - TextEmbeddingCreateRequest_parameters
+    - _TextEmbeddingCreateRequest_parameters
   TextEmbeddingLimit:
-    - TextEmbeddingLimitRetrieve_result
+    - _TextEmbeddingLimitRetrieve_result
   ConcurrencyLimit:
-    - TextGenerationLimitRetrieve_result_concurrency
-    - TextEmbeddingLimitRetrieve_result_concurrency
+    - _TextGenerationLimitRetrieve_result_concurrency
+    - _TextEmbeddingLimitRetrieve_result_concurrency
   TuneAssetType:
-    - TuneIdContentTypeRetrieveRequestParamsType
+    - _TuneIdContentTypeRetrieveRequestParamsType
   TextTokenizationParameters:
-    - TextTokenizationCreateRequest_parameters
+    - _TextTokenizationCreateRequest_parameters
   TextTokenizationReturnOptions:
-    - TextTokenizationCreateRequest_parameters_return_options
+    - _TextTokenizationCreateRequest_parameters_return_options
+  # OTHER PUBLIC SCHEMAS ---------------------------------------------------------
+  ModelIdRetrieveResult: [ _ModelIdRetrieve_result ]
```

Comment:
All schemas are private by default.

There is no need for a separate "public list", aliases already handle this

Reply:
I added quite a lot of publicly exposed schemas, because they are part of the response model (nested)
Do we need to expose them all? The only reason I can see is if user wants to use them for typing of the nested modles.